docs/tutorials/sentiment-analysis-on-french-tweets-using-mysql.mdx

```diff
@@ -0,0 +1,146 @@
+
+Once you added the details correctly, you get the `Saved Connection` popup.  In the MindsDB Web, the sidebar display list of databases. You would have created your database in `alias connection` , check for that name in the list and when you click on it, automatically the MindsDB queries the output. 
+
+```
+Select * 
+FROM mysql.french_tweets;
+```
+Your output would be like :
+![select query output](images/sentiment-analysis-on-french-tweets/query.png)
+
+## Load the Model
+
+We are focusing on text classification, we're going to use Hugging Face's [French Sentiment analysis model based on CamemBERT](https://huggingface.co/philschmid/pt-tblard-tf-allocine/tree/main) pytorch supported model. This Hugging Face transformer is a CamemBERT model trained on Allocine dataset.
+
+Our Hugging Face integration automatically manages downloading and deploying of pre-trained transformers from Hugging Face's hub. For example, we can download a transformer which has been trained to analyze the sentiment of text:
+```
+CREATE  MODEL mindsdb.sentiment_analyzer
+PREDICT sentiment
+USING
+engine  =  'huggingface',
+task =  'text-classification',
+model_name =  'philschmid/pt-tblard-tf-allocine',
+input_column =  'text';
+```
+To create a model in MindsDB, we first specify our chosen model name (sentiment_classifier) with **CREATE MODEL mindsdb.sentiment_analyzer.**
+
+Next, we choose the name of the column we would like to predict (sentiment) with **PREDICT sentiment**. Hugging Face models are already trained, so we don’t need to select a column from our training data - we can name our column whatever we like.
+
+Finally, we specify all the additional variables we need after **USING**. Each ML handler has slightly different parameters (for more details on Hugging Face you can check our [documentation](https://docs.mindsdb.com/custom-model/huggingface?_gl=1*dxyiz5*_ga*MjEyNzE5MjgzMy4xNjY4Nzg5OTkz*_ga_7LGFPGV6XV*MTY3MDkwMjEwOS41Mi4xLjE2NzA5MDI4ODYuNjAuMC4w).) In this case, we specify:  
+
+-   **engine = ‘huggingface’**  - the name of the ML framework to be used. Other options include ‘lightwood’ and ‘luwig’
+-   **model_name = 'philschmid/pt-tblard-tf-allocine'**  - the name of the model from  [Hugging Face](https://huggingface.co/models)
+-   **input_column = 'text'**  - the name of the input column that has your text.
+
+We currently support four of the Hugging Face NLP tasks: ‘text-classification’, ‘zero-shot-classification’, ‘translation’, and ‘summarization’. This model is ‘text-classification’.
+
+Once the above query is executed, we can check the status of the creation process:
+```
+SELECT  *
+FROM mindsdb.models
+WHERE  name='sentiment_analyzer';
+```
+After a few seconds you should see the status change from **generating** to **complete.**
+
+## Making predictions
+
+Once the status is complete, the behavior is the same as with any other AI table you can query it or by directly specifying your inputs, like so:
```

Comment:
Actually, you could include both single prediction and batch predictions as follows:
 
`...the behavior is the same as with any other AI table. You can query it directly by specifying your inputs:
SELECT  *  FROM mindsdb.sentiment_analyzer
WHERE  text='Je vais me crier après avoir regardé Marley et moi.';

Or by joining with the data table for batch predictions:
SELECT  *  FROM mindsdb.sentiment_analyzer as p
JOIN mysql.french_tweets as d;`